infra/cifuzz/external-actions/run_fuzzers/action.yml

```diff
@@ -0,0 +1,50 @@
+# action.yml
+name: 'run-fuzzers'
+description: 'Runs fuzz target binaries for a specified length of time.'
+inputs:
+  language:
+    description: 'Programming language project is written in.'
+    required: false
+    default: 'c++'
+  fuzz-seconds:
+    description: 'The total time allotted for fuzzing in seconds.'
+    required: true
```

Comment:
Hmm...I wonder what it means to default to 600 and be required. Maybe if user doesn't specify it will be set to 600 but it will always be set.

Reply:
Hmm, I'm not sure either. Seems like we can just remove "required" here. Let's leave that for another PR.